img/f/fragdenstaat.de.svg

```diff
@@ -0,0 +1 @@
+<svg width="456" height="399" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M76.4729 171.01v12.414l43.3501 25.812V171.01H76.4729ZM156.768 231.207l72.513 43.054 70.837-42.857v-44.335l-72.315 42.857-71.035-42.759v44.04ZM337.064 171.01v38.029l43.35-26.207V171.01h-43.35Z" fill="#296DFF"/><path d="M428 171.01v-37.931h-25.813L229.773 25 52.9261 133.079H28v37.931h11.5271v164.039H28v37.931h400v-37.931h-10.64V171.01H428Zm-127.882 0v164.039h-143.35V171.01h143.35ZM227.31 69.335l108.867 63.744H123.665L227.31 69.335ZM76.4729 171.01h43.3501v164.039H76.4729V171.01ZM380.414 335.049h-43.35V171.01h43.35v164.039Z" fill="#C9D2DC"/></svg>
```

Comment:
The padding is unnecessary. Here's a version without any padding/margin around the icon: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 348"><path fill="#296dff" d="M48.5 146v12.4L200 249.3l152.4-91.5V146h-43.3L200 205 91.8 146Z"/><path fill="#c9d2dc" d="M400 146v-38h-25.8L200 0 25 108H0v38h11.5v164H0v38h400v-38h-10.6V146Zm-127.9 0v164H128.8V146ZM200 44.3l108.2 63.8H95.7ZM48.5 146h43.3v164H48.5Zm304 164H309V146h43.3z"/></svg>
```